src/app/MenuView.ts

```diff
@@ -14,92 +15,81 @@ export class MenuView {
 
   displayMenu(): void {
     Render.removeAllChildren('#sf-app');
-    Render.render(
-      '#sf-app',
-      this.createLandingPage(),
-      this.createFooter(),
-    );
+    Render.render('#sf-app', this.createLandingPage());
   }
 
   private createLandingPage(): HTMLElement {
     return Render.elementFactory(
       'div',
       {
-        className: 'page__container menu-window',
+        className: 'page__container',
       },
-      this.createHeading(),
-      this.createPageContent(),
+      Render.elementFactory(
+        'div',
+        { className: 'page__container menu-window' },
+        this.createHeading(),
+        this.createPageContent(),
+        this.createFooter(),
+      ),
     );
   }
 
   private createHeading(): HTMLElement {
     return Render.elementFactory(
-      'header',
+      'h1',
       { className: 'menu-window__header' },
-      Render.elementFactory(
-        'h1',
-        { className: 'menu-window__heading heading' },
-        'FARM TYCOON',
-      ),
-      Render.elementFactory(
-        'h2',
-        { className: 'menu-window__description text' },
-        'Breed animals, Do your math, Be quick, Plan ahead, Protect your herd, Predators are there to get your animals! Become a farmer and be the first to gather all the animals!',
-      ),
+      'SUPERFARMER',
     );
   }
 
   private createPageContent(): HTMLElement {
     return Render.elementFactory(
       'div',
-      { className: 'menu' },
-      Render.elementFactory('div', {
-        className: 'menu__graphic-container',
-      }),
-      Render.elementFactory(
-        'div',
-        { className: 'menu__buttons' },
-        ...this.createMenuButtons(),
-      ),
-      Render.elementFactory('div', {
-        className: 'menu__graphic-container',
-      }),
-      Render.elementFactory(
-        'button',
-        {
-          className: 'menu__button--rules button',
-        },
-        'i',
-      ),
+      { className: 'menu__buttons' },
+      ...this.createMenuButtons(),
     );
   }
 
   private createMenuButtons(): HTMLElement[] {
-    const startGameButton = Render.elementFactory(
+    const rulesButton = Render.elementFactory(
       'button',
-      { className: 'button' },
-      'NEW GAME',
+      {
+        className: 'menu__buttons__button',
+      },
+      'GAME RULES',
     );
-    startGameButton.addEventListener('click', () => {
-      this.modeModal.showModal();
-    });
+    // rulesButton.setAttribute('disabled', 'true');
     const settingsButton = Render.elementFactory(
       'button',
       {
-        className: 'button',
+        className: 'menu__buttons__button',
```

Comment:
menu__button perhaps? Would make our scss structure less nested and easier to read